hack/gcp-service-account-maintenance/build-service-account.bash

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
```

Comment:
would it be possible to add `set -euo pipefail` for improved safety against unforeseen command failures?